doc/methsel.xml

```diff
@@ -100,6 +103,31 @@ function:
 <#Include Label="CallMethods">
 </Section>
 
+<Section Label="howstored">
+<Heading>Global records storing functions</Heading>
+
+The following global records store the methods for finding
+homomorphisms for group recognition.
+We collect them in these records such that we do not use up too many
+global variable names.
+
+<#Include Label="FindHomMethodsPerm">
+<#Include Label="FindHomMethodsMatrix">
+<#Include Label="FindHomMethodsProjective">
+<#Include Label="FindHomMethodsGeneric">
+
+This global record holds the functions for writing group
```

Comment:
This line is confusing. To what global record are you referring? And why only matrix group recognition.

Reply:
Ooh, yes. Thanks for spotting that.